app/scripts/keyring-controller.js

```diff
@@ -171,11 +171,8 @@ module.exports = class KeyringController extends EventEmitter {
   //
   // Used when creating a first vault, to allow confirmation.
   // Also used when revealing the seed words in the confirmation view.
-  placeSeedWords () {
-    const hdKeyrings = this.keyrings.filter((keyring) => keyring.type === 'HD Key Tree')
-    const firstKeyring = hdKeyrings[0]
-    if (!firstKeyring) throw new Error('KeyringController - No HD Key Tree found')
-    return firstKeyring.serialize()
+  placeSeedWords (selectedKeyring) {
```

Comment:
TBH this shouldnt really be in the KrC at all. More soon-to-be-deprecated "primary account" stuff.